app/controllers/candidate_interface/application_form_controller.rb

```diff
@@ -26,7 +26,9 @@ def submit
       end
     end
 
-    def submit_success; end
+    def submit_success
+      @application_ref = current_candidate.current_application.reference
```

Comment:
Calling it `.reference` immediately made me think of `references`... I think a different name would be useful. `application_reference` is more verbose but in this situation I think it's better?

Reply:
after an off-line chat, we decided for `support_reference`